packages/sku/scripts/postinstall.js

```diff
@@ -1,12 +1,7 @@
 #!/usr/bin/env node
-// @ts-check
+// @ts-nocheck
```

Comment:
IMO it's still worth typechecking this code, even if there's now less of it and no jsdoc stuff.